src/Platforms/OraclePlatform.php

```diff
@@ -841,12 +847,13 @@ public function usesSequenceEmulatedIdentityColumns(): bool
 
     public function getIdentitySequenceName(string $tableName, string $columnName): string
     {
-        $table = new Identifier($tableName);
+        $table = $this->normalizeIdentifier($tableName);
+        $column = $this->normalizeIdentifier($columnName);
 
-        // No usage of column name to preserve BC compatibility with <2.5
-        $identitySequenceName = $this->addSuffix($table->getName(), '_SEQ');
+        $colname = $columnName ? '_' . $column->getName() : '';
+        $identitySequenceName = $this->addSuffix($table->getName() . $colname, '_seq');
```

Comment:
Why does the column name need to be part of the sequence name? Can a table have multiple auto-increment columns?

Reply:
Yes, tables can have multiple sequences in Oracle and PostgreSQL compared to only one autoincrement column in MySQL. That's also the reason why the DBAL PostgreSQL platform code creates sequences names as combination of table and column name.